registry/src/pages/createNamespace.js

```diff
@@ -15,6 +17,33 @@ const NamespaceForm = () => {
   const message = useSelector((state) => state.createNamespace.message);
   const statuscode = useSelector((state) => state.createNamespace.statuscode);
 
+  useEffect(() => {
+    if (statuscode === 200) {
+      toast.success(message, {
+        position: "top-center",
+        autoClose: 5000,
+        hideProgressBar: false,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: "light",
+      });
+    }
+    if (statuscode !== 0 && statuscode !== 200) {
```

Comment:
How about having just an `else` condition ?

Reply:
else would work at every reload of the page. ( thus show a toast at every reload with an error)